samples/sensor/lis2dh/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _lis2dh:
+
+LIS2DH: Accelerometer Monitor
+#############################
+
+Overview
+********
+
+This sample application periodically reads accelerometer data from the
+LIS2DH sensor, and displays the sensor data on the console.
+
+Requirements
+************
+
+This sample uses the LIS2DH, ST MEMS system-in-package featuring a 3D
+digital linear acceleration sensor, controlled using the I2C interface.
+
+References
+**********
+
+For more information about the LIS2DH eCompass module, see
+http://www.st.com/en/mems-and-sensors/lis2dh.html
+
+Building and Running
+********************
+
+This project outputs sensor data to the console. It requires a LIS2DH
+system-in-package, which is present on the stm32f3_disco board.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/lis2dh
+   :board: stm32f3_disco
+   :goals: build
+   :compact:
+
```

Comment:
I see the build part, but what about the flashing and running?  Don't forget to mention you'll need to connect the board to your host computer and run a serial terminal program.

Reply:
Huh.  Copied that and most of the rest of the sample from lsm6dsl which apparently doesn't have that material.

Reply:
@dbkinder Many if not most (sensor) samples have no more documentation than this; e.g.:

* https://docs.zephyrproject.org/latest/samples/sensor/adt7420/README.html
* https://docs.zephyrproject.org/latest/samples/sensor/bme680/README.html
* https://docs.zephyrproject.org/latest/samples/sensor/ens210/README.html

If there is a new expectation for how samples are to be documented, could you point me to a template I can copy?  Thanks.

Reply:
Since you asked, there is a sample template in ``doc/templates/sample.tmpl``.  There's a comment in the samples/index.rst that points this template out but it's not obvious.

Sadly, not all samples follow this template, particularly the sensor samples that punt the issue with statements such as "Flash the binary to a board of choice with a sensor connected." but don't provide instructions on how to connect the sensor (or how to flash).

As we get new developers trying out Zephyr, we're getting more comments about incomplete documentation that I'm hoping we can resolve as new documentation shows up and we can eventually address in the documentation that's already included.  If you'd like to help fix this situation, I'd appreciate your adding those instructions to your PR (and even others if you can).  I'll approve this PR with those instructions missing for now if you're too busy.